build.gradle

```diff
@@ -7,6 +7,11 @@ plugins {
     id 'maven-publish'
     id 'signing'
     id "com.google.protobuf" version "0.9.3"
+    id "io.github.gradle-nexus.publish-plugin" version "1.3.0" apply(false)
+}
+
+if (project == rootProject) {
```

Comment:
This plugin requires that it is in the root project. When the Test Harness runs for a GITHUB SHA or in LOCAL_MODE it treats the java-server-sdk as a sub-project and this publish plugin will complain and block gradle from running. Since we don't care about publishing in the test harness I wrapped the problematic areas with this statement so that they aren't loaded when java-server-sdk is used this way